doc/select.md

```diff
@@ -0,0 +1,204 @@
+# Select examples
+
+## Filtering
+
+The second parameter passed to ``crud.select`` is an array of conditions. Below are examples of filtering data using these conditions. 
+
+### Getting full space
+
+To get a full space without filtering, you need to pass ``nil`` as a condition.
+
+**Example:**
+
+```lua
+crud.select('customers', nil)
+---
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucked_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'surname', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [1, 635, 'John', 'Adams', 65]
+  - [2, 2364, 'John', 'Sidorov', 28]
+  - [3, 6517, 'Ronald', 'Dump', 77]
+  - [4, 563, 'Sergey', 'Lee', 21]
+  - [5, 2313, 'Tatyana', 'May', 20]
+...
+```
+
+### Select using index
+
+Let's say we have a ``age`` index. Example below gets a list of ``customers`` over 30 years old.
+
+**Example:**
+
+```lua
+res, err = crud.select('customers', {{'>=', 'age', 30}})
+res.rows
+---
+- - [1, 635, 'John', 'Adams', 65]
+  - [3, 6517, 'Ronald', 'Dump', 77]
+...
+```
+
+### Select using composite index
+
+Suppose we have a composite index consisting of the ``name`` and ``surname`` fields. See example of select queries using such a composited index below.
+
+**Example**:
+
+```lua
+res, err = crud.select('customers', {{'==', 'full_name', {"John", "Sidorov"}}})
+res.rows
+---
+- - [1, 2364, 'John', 'Sidorov', 28]
+...
+```
+
+### Select using partial key 
+
+Alternatively, you can use a partial key for a composite index.
+
+**Example**:
+
+```lua
+res, err = crud.select('customers', {{'==', 'full_name', "John"}})
+res.rows
+---
+- - [1, 2364, 'John', 'Sidorov', 28]
+  - [2, 635, 'John', 'Adams', 65]
+...
+```
+
+### Select using non-indexed field
+
+You can also make a selection using a non-indexed field.
+
+**Example:**
+
+```lua
+res, err = crud.select('customers', {{'>=', 'id', 3}})
+res.rows
+---
+- - [3, 6517, 'Ronald', 'Dump', 77]
```

Comment:
Id `id` a non-indexed field?

Reply:
Usually, ID is a primary index.
Maybe we should show space indexes before all examples.

BTW, you have `surname` field that is non-indexed. And it can show that the different between using first and other parts of composite index (`name` vs `surname`).